lib/repo-type-mappings/yocto-based-OS-image/versionist.conf.js

```diff
@@ -59,7 +59,7 @@ module.exports = {
       return semver.inc(currentVersion, 'patch')
     }
     const parsedCurrentVersion = semver.parse(currentVersion)
-    if ( parsedCurrentVersion.build ) {
+    if ( ! _.isEmpty(parsedCurrentVersion.build) ) {
```

Comment:
```suggestion
    if ( parsedCurrentVersion.build && parsedCurrentVersion.build.length ) {
```
Specifically `parsedCurrentVersion.build` should be an array, correct? I'm not familiar with using `isEmpty` to check an array but that doesn't mean it's wrong!!

Reply:
@klutchell `_.isEmpty` works on arrays, see https://lodash.com/docs/4.17.15#isEmpty.

Is it preferred not to use isEmpty?

Reply:
> Is it preferred not to use isEmpty?

No idea what's preferred, I just haven't used Lodash much myself

I'm sure what you have is good enough or we can get another opinion?

Reply:
I am re-running the tests (which I skipped last time as I was convinced the change was mostly aesthetic), and I will add an extra reviewer.

Reply:
Hard to tell what existing code expects without types... But if `isEmpty` returns `false` on `undefined` we're probably covered.

Reply:
thanks @stathismor, it does. I will finish my testing and undraft.